00-intro/intro.md

```diff
@@ -15,39 +15,33 @@ links-as-notes: true
 
 ## Overview
 
-\Gls{NeoFS} is a decentralized distributed object storage system integrated with
-the [Neo Blockchain](https://neo.org).
+\Gls{NeoFS} is a decentralized distributed object storage system integrated with [Neo Blockchain](https://neo.org).
 
-\Glspl{Node} are organized in peer-to-peer network that takes care of storing
-and distributing user's data. Any Neo user may participate in the network and
-get paid for providing storage resources to other users or store his data in
-NeoFS and pay a competitive price for it.
+We store and distribute users' data across a peer-to-peer network of \Glspl{Node}. Whether a business or an individual, any Neo user may join the network and get paid for providing storage resources to others or pay a competitive price and enjoy NeoFS storage solution.
 
-Users can reliably store object data in the NeoFS network and have a transparent
-data placement process due to decentralized architecture and flexible storage
-policies. Under the users we mean both individuals and enterprises. Each
-\Gls{Node} is responsible for executing the storage policies that the users
-select for geographical location, reliability level, number of nodes, type of
-disks, capacity, etc. Thus, NeoFS gives full control over data to users.
+The decentralized architecture and flexible storage
+policies allow users to reliably store object data in the NeoFS network. Each \Gls{Node} is responsible for executing the storage policies selected by the users
+regarding the geographical location, reliability level, number of nodes, type of
+disks, capacity, etc. Thus, NeoFS enables a transparent
+data placement process and gives full control over data to the users.
 
 Deep [Neo Blockchain](https://neo.org) integration allows NeoFS to be used by
 \glspl{dApp} directly from
 [NeoVM](https://docs.neo.org/docs/en-us/basic/technology/neovm.html) on the
 [Smart
 Contract](https://docs.neo.org/docs/en-us/basic/technology/neocontract.html)
-code level. This way dApps are not limited to on-chain storage and can
+code level. As a result, dApps are not limited to on-chain storage and one can
 manipulate large amounts of data without paying a prohibitive price.
 
-NeoFS has native [gRPC](https://grpc.io) \Gls{api} and popular protocol gates
+NeoFS uses native [gRPC](https://grpc.io) \Gls{api} and popular protocols
```

Comment:
Here we need to empathize that different protocol support is implemented as a separate entities, called protocol gates. Those are separate software, run and maintained aside of the main NeoFS network. Is there a way to describe it here in a short form?

Reply:
Are you sure that it is called protocol gate in English? I could find almost no examples of this phrase in the Internet.

Reply:
Agree. What do you think about `protocol gateway` then?

Reply:
"A **protocol gateway** is a device that converts from one protocol to another to allow communication between devices". That's what I've found. If it's correct, let it be protocol gateway.